app/views/spree/shared/_nav_bar.html.erb

```diff
@@ -1,7 +1,70 @@
-      <%= render partial: 'spree/shared/search' %>
+      <%#= render partial: 'spree/shared/search' %>
     </li>
   </ul>
-</nav>
+</nav>  
+   -->
+      <div class="col-xl-10 position-static">
+        <div class="navbar-collapse">
+          <ul class="navbar-nav tablet-hide">
+            <li class="nav-item"><a class="nav-link" href="#">Extracts &amp; Flavorings</a>
+              <div class="dropdown-menu-wrap">
+                <div class="row no-gutters">
+                  <div class="col-lg-2"></div>
+                  <div class="col-lg-7 has-gridlines">
+                    <div class="row no-gutters">
+                      <div class="col">
+                        <ul class="dropdown-menu d-flex flex-column">
+                          <li><a href="#">Flavor Extracts<span class="badge popular-badge">Popular</span></a></li>
+                          <li><a href="#">Flavor Concentrate</a></li>
+                          <li><a href="#">Flavor Compounds</a></li>
+                          <li><a href="#">Flavor Emulsions</a></li>
+                          <li><a href="#">Flavor Powsers<span class="badge popular-badge">Popular</span></a></li>
+                          <li><a href="#">Flavor Oils OILS</a></li>
+                        </ul>
+                      </div>
+                      <div class="col">
+                        <ul class="dropdown-menu d-flex flex-column">
+                          <li><a href="#">Vanilla Extracts</a></li>
+                          <li><a href="#">Flavoring for Ice Cream</a></li>
+                          <li><a href="#">Flavoring for Water</a></li>
+                          <li><a href="#">Flavoring for Coffee &amp; Tea</a></li>
+                          <li><a href="#">Flavoring for Lip Balm</a></li>
+                          <li><a href="#">Flavoring for Chocolate</a></li>
+                        </ul>
+                      </div>
+                      <div class="col">
+                        <ul class="dropdown-menu d-flex flex-column">
+                          <li><a href="#">Keto Friendly Flavors</a></li>
+                          <li><a href="#">TTB Approved Flavors</a></li>
+                          <li><a href="#">Flavors without Diacetyl</a></li>
+                          <li><a href="#">Flavors sample packs</a></li>
+                        </ul>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="col-lg-3"></div>
+                </div>
+              </div>
+            </li>
+            <li class="nav-item"><a class="nav-link" href="#">Fragrances</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">Syrups</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">Baking Ingredients</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">More</a></li>
+            <li class="nav-item">
+              <% if spree_current_user.present? %>
```

Comment:
Please remove this new nav item  here  and add login and logout check on this item
<img width="83" alt="Screen Shot 2021-11-08 at 12 20 02 PM" src="https://user-images.githubusercontent.com/78048076/140699740-f4a48873-697f-4a97-9324-f11a17d95ff5.png">